client_integration_darwin_test.go

```diff
@@ -0,0 +1,41 @@
+package sftp
```

Comment:
I found that the field names available in syscall.Statfs_t were adequately different between kernels (darwin vs linux) that I could not re-use client_integration_linux_test.go when testing on darwin.